app/Filament/Resources/UserResource/Pages/ListUsers.php

```diff
@@ -36,18 +36,40 @@
 
 namespace App\Filament\Resources\UserResource\Pages;
 
-use Filament\Actions;
+use App\Models\User;
+use App\Settings\LicenseSettings;
+use Filament\Actions\CreateAction;
+use Illuminate\Support\HtmlString;
 use App\Filament\Resources\UserResource;
 use Filament\Resources\Pages\ListRecords;
+use Illuminate\Contracts\Support\Htmlable;
 
 class ListUsers extends ListRecords
 {
     protected static string $resource = UserResource::class;
 
+    public function getSubheading(): string | Htmlable | null
+    {
+        $count = User::count();
+        $max = app(LicenseSettings::class)->data->limits->crmSeats;
+
+        $colorClass = 'text-green-500';
+
+        if ($count > $max) {
+            $colorClass = 'text-red-500';
+        } elseif ($count === $max) {
+            $colorClass = 'text-yellow-500';
+        }
+
+        return new HtmlString(
+            "<span class='{$colorClass}'>{$count}/{$max}</span> CRM Seats Used"
+        );
+    }
```

Comment:
IMO this should be extracted into a Blade view as it contains HTML.

Also, we should probably use `success`, `danger` and `warning` colors instead